File: Nhom02LapTrinhMobile/WebServer/WebsiteBanHang_VoNguyenDuyTan_2001200082/Model/Topping.php
<?php
class Topping
{
    public $MaTopping;
    public $TenTopping;
    public $Gia;
    public $HinhAnh;

    public function getMaTopping()
    {
        return $this->MaTopping;
    }
    public function setMaTopping($MaTopping)
    {
        $this->MaTopping=$MaTopping;
    }
    public function getTenTopping()
    {
        return $this->TenTopping;
    }
    public function setTenTopping($TenTopping)
    {
        $this->TenTopping=$TenTopping;
    }
    public function getGia()
    {
        return $this->Gia;
    }
    public function setGia($Gia)
    {
        $this->Gia=$Gia;
    }
    public function getHinhAnh()
    {
        return $this->HinhAnh;
    }
    public function setHinhAnh($HinhAnh)
    {
        $this->HinhAnh=$HinhAnh;
    }
    public function __construct($MaTopping="",$TenTopping="",$Gia=0,$HinhAnh="")
    {
        $this->HinhAnh=$HinhAnh;
        $this->MaTopping=$MaTopping;
        $this->Gia=$Gia;
        $this->TenTopping=$TenTopping;
    }

    // public static function getAll($values)
    // {
    //     $ListTopping=array();
    //     foreach($values as $value)
    //     {
    //         $topping=new Topping();
    //         $topping->setMaTopping($value["MaTopping"]);
    //         $topping->setTenTopping($value["TenTopping"]);
    //         $topping->setGia($value["Gia"]);
    //         $topping->setHinhAnh($value["HinhAnh"]);
    //         $ListTopping[]=$topping;
    //     }
    //     return $ListTopping;
    // }


    public static function getAll()
    {
        $ListTopping=array();
        $sql="select*from topping";
        $connection=new Connection();
        $resultData=$connection->excuteQuery($sql);
        if($resultData!=null)
        {
            foreach($resultData as $value)
            {
                $topping=new Topping();
                $topping->setMaTopping($value["MaTopping"]);
                $topping->setTenTopping($value["TenTopping"]);
                $topping->setGia($value["Gia"]);
                $topping->setHinhAnh($value["HinhAnh"]);
                $ListTopping[]=$topping;
            }
            return $ListTopping;
        }
        return null;
    }

     public static function getOneByID($MaTopping)
    {
        $sql="SELECT *from topping WHERE MaTopping=$MaTopping";
        $connection=new Connection();
        $resultData=$connection->excuteQuery($sql);
        if($resultData!=null)
        {
            foreach($resultData as $value)
            {
                $topping=new Topping();
                $topping->setMaTopping($value["MaTopping"]);
                $topping->setTenTopping($value["TenTopping"]);
                $topping->setGia($value["Gia"]);
                $topping->setHinhAnh($value["HinhAnh"]);
                return $topping;
            }
        }
        return null;
    }
}



?>

File: Nhom02LapTrinhMobile/AppAndorid/Nhom02_BanTraSua/app/src/main/java/com/example/nhom02_bantrasua/Model/Topping.java
package com.example.nhom02_bantrasua.Model;

public class Topping {
    public String MaTopping;
    public String TenTopping;
    public double Gia;
    public String HinhAnh;

    public String getMaTopping() {
        return MaTopping;
    }

    public void setMaTopping(String maTopping) {
        MaTopping = maTopping;
    }

    public String getTenTopping() {
        return TenTopping;
    }

    public void setTenTopping(String tenTopping) {
        TenTopping = tenTopping;
    }

    public double getGia() {
        return Gia;
    }

    public void setGia(double gia) {
        Gia = gia;
    }

    public String getHinhAnh() {
        return HinhAnh;
    }

    public void setHinhAnh(String hinhAnh) {
        HinhAnh = hinhAnh;
    }
    public Topping() {
        MaTopping = "";
        TenTopping = "";
        Gia = 0;
        HinhAnh = "ThachCaPhe.png";
    }
    public Topping(String maTopping, String tenTopping, double gia, String hinhAnh) {
        MaTopping = maTopping;
        TenTopping = tenTopping;
        Gia = gia;
        HinhAnh = hinhAnh;
    }

}
